media/css/mozorg/newsletter-test.less

```diff
@@ -0,0 +1,202 @@
+            .background-size(120px, 120px);
+            .span_narrow(4);
+            background-position: left 10px;
+            padding-left: 140px;
+            margin-left: -((@gridColumnWidthNarrow * 7) + (@gridGutterWidth * 6));
+            margin-top: -10px;
+
+            h3 {
+                .font-size(32px);
+            }
+
+            h4 {
+                .font-size(20px);
+            }
+        }
+
+        @media (max-width: @breakTablet) {
+            .background-size(100px, 100px);
+            .span-all;
+            background-position: center top;
+            margin-top: 0;
+            min-height: 0;
+            padding-left: 0;
+            padding-top: 120px;
+            text-align: center;
+
+            h3 {
+                .font-size(32px);
+            }
+        }
+    }
+
+    .newsletter-form {
+        .span(5);
+        .offset(7);
+        padding-top: 20px;
+
+        .form-submit,
+        .form-contents {
+            .span-all;
+        }
+
+        @media (max-width: @breakDesktop) {
+            .span_narrow(5);
+            .offset_narrow(7);
+        }
+
+        @media (max-width: @breakTablet) {
+            .span-all;
+        }
+    }
+
+    .field-format label {
+        display: inline;
+        margin-right: 20px;
+    }
+
+    input[type='email'] {
+        .border-box;
+        .border-radius(4px);
+        .box-shadow(none);
+        .font-size(@largeFontSize);
+        border: 0;
+        padding: .75em 20px;
+        width: 100%;
+        height: auto;
+        background: #fff;
+    }
+
+    select {
+        .font-size(@baseFontSize);
+        font-family: inherit;
+    }
+
+    .button {
+        .border-box;
+        .border-radius(5px);
+        .box-shadow(none);
+        .font-size(@largeFontSize);
+        .transition(background-color .15s ease);
+        background: transparent;
+        border: 2px solid #fff;
+        padding: .75em 20px;
+        text-shadow: none;
+        text-transform: uppercase;
+        width: 100%;
+
+        &:hover,
+        &:focus {
+            background: #33aae4;
+            background: rgba(255, 255, 255, .1);
+        }
+    }
+
+    a {
+        color: #fff;
+        text-decoration: underline;
+    }
+
+    #footer-email-errors {
+        .span-all;
+    }
+
+    #newsletter-form-thankyou {
+        color: #fff;
+
+        h3 {
+            color: #fff;
+        }
+    }
+}
+
+#home {
+    #newsletter-subscribe {
+        margin-bottom: 0;
+
+        h3 {
+            padding: 0;
+            text-align: left;
+
+            @media (max-width: @breakTablet) {
+                text-align: center;
+            }
+        }
+
+        ::-webkit-input-placeholder {
+           color: #ccc;
+           font-weight: normal;
+        }
+
+        ::-moz-placeholder {
+           color: #ccc;
+           font-weight: normal;
+        }
+
+        :-ms-input-placeholder {
+           color: #ccc;
+           font-weight: normal;
+        }
```

Comment:
Most clients, when encountering a rule with an unknown selector, will ignore the entire rule. So for vendor prefixed selectors like this you have to repeat the rule for each vendor. Kinda sucks.